### PR TITLE
schedulers: unify hot region scheduler's influence

### PR DESCRIPTION
**Pull request by Yisaer**
Signed-off-by: yisaer <disxiaofei@163.com>

### What problem does this PR solve?
Currently, the pending influence holding by hot region scheduler is classified by the `resourceType`, not `read/write type`.
This will cause the write-peer and write-leader can't see each other's influence(either do read-peer and read-leader in the future)

### What is changed and how it works?
1. unify hot region scheduler's influence
2. make read/write pending influence can be summarized by each hot schedule operator

### Check List

Tests

- Unit test

### Release note

* No release not

### CODE REVIEW

**Review by lhy1024 — COMMENTED** [review]
It will influence correctly between read and write. But I think unified pending influence will be better.

**Review by lhy1024 — APPROVED** [review]

File: server/schedulers/hot_region.go [hunk 1]
```diff
@@ -1099,19 +1093,18 @@ func (h *hotScheduler) GetHotWriteStatus() *statistics.StoreHotPeersInfos {
...
+	return h.copyPendingInfluence()
 }
 
 func (h *hotScheduler) GetReadPendingInfluence() map[uint64]*Influence {
-	return h.copyPendingInfluence(readLeader)
+	return h.copyPendingInfluence()
 }
 
-func (h *hotScheduler) copyPendingInfluence(ty resourceType) map[uint64]*Influence {
+func (h *hotScheduler) copyPendingInfluence() map[uint64]*Influence {
```
Comment (rleungx) [inline server/schedulers/hot_region.go:1103]: directly use it?

**Review by rleungx — COMMENTED** [review]
The rest LGTM.

**Review by lhy1024 — APPROVED** [review]

**Review by rleungx — APPROVED** [review]

**Pull request merged**